feature/experimental/isis/ate_tests/isis_interface_passive_test/README.md

```diff
@@ -0,0 +1,96 @@
+## RT-2.7: IS-IS Passive is enabled at interface level
```

Comment:
this line should only start with a single #. The ## is causing the addrundata check to fail

Reply:
Addressed in latest commit